### PR TITLE
Reactivating fenics

### PR DESCRIPTION
**Pull request by nrichart**
depends on https://github.com/epfl-scitas/spack/pull/253

### CODE REVIEW

File: humagne.yaml [hunk 1]
```diff
@@ -566,7 +571,12 @@ packages:
...
-      # TODO: - fenics@2017.1.0+petsc+hdf5~trilinos+suite-sparse+mpi ^petsc@3.7.6+boost+double+hdf5+metis+mpi+mumps~superlu-dist~hypre ^hdf5+szip+mpi~cxx  ^mumps@5.1.1+mpi+parmetis+metis+scotch+ptscotch ^parmetis@4.0.3 ^metis@5.1.0+real64 ^scotch@6.0.6+esmumps+mpi~metis ^netlib-scalapack@2.0.2 ^boost@1.61.0+icu_support+mpi+python ^python@2.7.13 ^eigen@3.3.1~debug+fftw+metis+mpfr+scotch+suitesparse ^fftw@3.3.6-pl2+mpi~openmp
+      - fenics@2019.1.0+petsc+hdf5~trilinos+suite-sparse+mpi~slepc
+        ^petsc~int64+double+hdf5+metis+mpi+mumps+superlu-dist+hypre+suite-sparse
+        ^hdf5+szip+mpi+hl+fortran+cxx
+        ^mumps+mpi+parmetis+metis+scotch+ptscotch
+        ^parmetis ^metis@5.1.0+real64 ^scotch@6.0.6+esmumps+mpi~metis
+        ^python@3.7.3+optimizations+tkinter
       # FIXME: - simpson@master ^gsl@2.1 ^nfft@3.3.2 ^fftw@3.3.4 ^tk@8.6.3 ^tcl@8.6.5
 
   mpi-lapack-applications:
```
Comment (rmsds) [inline humagne.yaml:570]: Shouldn't fenics just be once under here? Instead of on the gnu/intel-specific sections?
Both specs look identical.

**Review by rmsds — CHANGES_REQUESTED** [review]
And I think we also need to add `fenics` to the whitelist in the `modules.yaml`.
(Or maybe we don't need it as it is explicit?)

**Comment by nrichart** [comment]
It should be fine for the module it should not be in a part that is blacklisted

**Review by rmsds — APPROVED** [review]
LGTM

**Pull request merged**